plugins/org.polarsys.capella.vp.ms.expression.ag/src/org/polarsys/capella/vp/ms/expression/ag/SituationExpressionExportHandler.java

```diff
@@ -89,6 +83,7 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
     try {
       exporter.export(toExport, columnOrder, bytes);
       ByteArrayInputStream inbb = new ByteArrayInputStream(bytes.toByteArray());
+      file.delete(false, new NullProgressMonitor());

```

Comment:
Not sure, but raise an CoreException when the file doesn't exist, which make the command doesn't work when the file doesnt exist) no?
maybe a if (file.exists())

Reply:
Fixed.
Also renamed commit to reference #issue.